### DISCUSSION TITLE
MIMIC-IV - Antibiotics

### DISCUSSION THREAD

**Discussion by aoifeboyle3004** [MIT-LCP/mimic-code#1681] [MIMIC-IV]
Is there anyway of identifying class of drugs for prescription table- want to pull all patients in ICU >48hours on antibiotics. Using prescription tbl for antibiotics but all character entries for drug so unsure how to filter for only antibiotics? Don't want to use inputevents (has antibiotics category) as aware potential for missing some that not included in ICU profile

**Comment by alistairewj** [comment]
I ended up writing a (really inelegant) case statement using string matching for all the antibiotics I could think of: https://github.com/MIT-LCP/mimic-code/blob/main/mimic-iv/concepts/medication/antibiotic.sql

Obviously would be much better if it classified the antibiotics into some useful ontology, but that is usually a fairly labor intensive mapping process.

**Reply by aoifeboyle3004** [reply]
Thank you this was extremely helpful!
